src/backend/InvenTree/part/serializers.py

```diff
@@ -1558,6 +1558,8 @@ class BomItemSerializer(
 
     import_exclude_fields = ['validated', 'substitutes']
 
+    export_exclude_fields = ['substitutes']
```

Comment:
Q (nb): why are these not included? is that an issue with nesting?

Reply:
Substitutes is a list of primary keys which does not export properly in the default exporter. However, it is now handled in the custom BOM exporter - multiple columns added for each substitute part.